dsls/sql/src/main/java/org/apache/beam/dsls/sql/example/BeamSqlExample.java

```diff
@@ -58,16 +60,31 @@ public static void main(String[] args) throws Exception {
     PCollection<BeamSQLRow> inputTable = PBegin.in(p).apply(Create.of(row)
         .withCoder(new BeamSqlRowCoder(type)));
 
-    //run a simple SQL query over input PCollection;
-    String sql = "select c2, c3 from TABLE_A where c1=1";
-    PCollection<BeamSQLRow> outputStream = inputTable.apply(BeamSql.simpleQuery(sql));
+    //Case 1. run a simple SQL query over input PCollection with BeamSql.simpleQuery;
+    PCollection<BeamSQLRow> outputStream = inputTable.apply(
+        BeamSql.simpleQuery("select c2, c3 from TABLE_A where c1=1"));
```

Comment:
@XuMingmin Having the ability to use `'SELECT C1, C2 FROM TABLE_NAME'` or `'SELECT C1, C2'` doesn't matter to me. What matters to me is that they don't need to use a `PCollectionTuple` with a single `PCollection` and `TupleTag` pair.

I was thinking that the whole idea of global registration whenever someone calls `query` or `simpleQuery` will impact the users future pipeline construction depending on the order in which they apply parts of their pipeline is not a good idea. I can see how its useful in a CLI where they aren't building a pipeline programmatically but should be limited to paths which the CLI code handles.